DESCRIPTION

```diff
@@ -27,10 +27,9 @@ Imports:
     ggplot2 (>= 3.3.0),
     reactable (>= 0.4.0)
 Suggests:
+    Benchmarking (>= 0.30),
```

Comment:
For the moment, we still need to import Benchmarking, but this will be fixed in the next PR when we update the report and the bootstrapping feature.

Reply:
If we update the report we can move it to Suggets. All application code, incl. bootstrapping, now relies on our own custom functions.